sdk/logical/system_view.go

```diff
@@ -176,6 +180,10 @@ func (d StaticSystemView) LookupPluginVersion(_ context.Context, _ string, _ con
 	return nil, errors.New("LookupPluginVersion is not implemented in StaticSystemView")
 }
 
+func (d StaticSystemView) ListVersionedPlugins(_ context.Context, _ consts.PluginType) ([]pluginutil.VersionedPlugin, error) {
+	return nil, errors.New("ListVersionedPlugins is not implemented in StaticSystemView")
```

Comment:
Is there a reason (besides matching surrounding style) to use `errors.New()` here and `fmt.Errorf()` in other places? Pretty minor detail, but just something that occurred to me when comparing this to https://github.com/hashicorp/vault/pull/16982/files#diff-321fc062fce4a794ac87052c0863ebd4177ac8e21d9cb1b718bf21c1851b5febR264

Reply:
My personal preference is for `errors.New` when there are no format arguments, but functionally they're the same, and linters will catch mismatches so I just chose to match the local style.